content.js

```diff
@@ -17,14 +17,18 @@ function defaultTheme() {
 
 if (!window.isSecureContext) {
   chrome.storage.local.get({
-    theme: "default"
+    theme: "default",
+    exceptions: ""
   }, function(items) {
     var theme = items.theme;
+    var exceptions = items.exceptions.split(',')
```

Comment:
However, I'm a little concerned about parsing an exception list on every page load. I don't think you can store `Set`s in `chrome.storage`, but could you store it as a map?

    {
      "bing.com": true
      "example.com": true
    }